packages/jahia/migrations/migrate-to-v13.yaml

```
@@ -0,0 +1,281 @@
+  - setEnvVersion: ${globals.version}
+  - logEvent:
+      target: ${nodes.proc.first.id}
+      title: "Environment $envName migrated"
+      text: "Environment $envName migrated to v${globals.version}"
+  - checkJahiaHealth: "cp, proc"
+  - checkJahiaDatadogCustomChecks: "cp, proc"
+  - checkHaproxyHealth: bl
+  - checkHaproxyDatadogCustomChecks: bl
+  - checkMariadbHealth: sqldb
+  - checkGaleraClusterHealth: sqldb
+  - checkMariadbDatadogCustomChecks: sqldb
+  ### End Post-actions
+
+actions:
+  eventsUpdate:
+    install:
+      jps: "${baseUrl}../update-events.yml"
+
+  setupPat:
+    - set:
+        samModuleName: server-availability-manager
+        patModuleName: personal-api-tokens
+
+    - cmd[bl]: |- # make haproxy check target /ping.jsp instead of /modules/healthcheck
+        haproxy_jahia_conf="/etc/haproxy/haproxy.cfg.jahia/jahia-cloud.cfg"
+        sed -i -e 's/\(option httpchk GET \).*\( "HTTP.*\)/\1"\/ping.jsp"\2/g' $haproxy_jahia_conf
+        sed -i -e 's/\(http-check expect rstring \).*/\1PONG/g' $haproxy_jahia_conf
+        service haproxy reload
+      user: root
+
+    - getJahiaVersion
+    - enableKarafLogin: proc
+    - removeHealthcheckModule
+
+    - isVersionHigherOrEqual:
+        a: ${globals.jahiaVersion}
+        b: 8.1.0.0
+        res: jahia8
+
+    - if ( ${globals.jahia8}):
+        - isVersionStrictlyLower:
+            a: ${globals.jahiaVersion}
+            b: 8.1.0.0
+            res: jahia80
+        - if ( ${globals.jahia80}):
+            installModuleVersionAndRemoveOthers:
+              moduleName: graphql-dxm-provider
+              version: 2.6.0
+
+        - installModuleVersionAndRemoveOthers:
+            moduleName: ${this.patModuleName}
+            version: 1.1.0
+        - installModuleVersionAndRemoveOthers:
+            moduleName: ${this.samModuleName}
+            version: 2.1.0
+    - else: # Jahia 7
+        - isVersionHigherOrEqual:
+            a: ${globals.jahiaVersion}
+            b: 7.3.9.0
+            res: jahia739OrHigher
+        - if (${globals.jahia739OrHigher}):
+            set:
+              samVersion: 1.1.0
+        - else: # jahia < 7.3.9
+            - installModuleVersionAndRemoveOthers:
+                moduleName: graphql-dxm-provider
+                version: 1.10.0
+            - set:
+                samVersion: 1.0.1
+
+        - installModuleVersionAndRemoveOthers:
+            moduleName: ${this.patModuleName}
+            version: 0.5.0
+        - installModuleVersionAndRemoveOthers:
+            moduleName: ${this.samModuleName}
+            version: ${this.samVersion}
+
+    - cmd[cp, proc]: |-
+        echo "probes.PatchFailures.severity=HIGH" >> /data/digital-factory-data/karaf/etc/org.jahia.modules.sam.healthcheck.ProbesRegistry.cfg
+
+    - getVaultData
+    - vaultDoesSecretExistsOrIsReadable:
+        secretPath: "paas/customers/${globals.organizationName}/paas_${env.shortdomain}/PAT/root"
+    - if ( ! ${globals.secretExistsOrIsReadable}):
```

_please select:_
```suggestion
    - if (! ${globals.secretExistsOrIsReadable}):
```
_or_
```suggestion
    - if ( ! ${globals.secretExistsOrIsReadable} ):
```
:trollface:

---

done